inst/tests/tests.Rraw

```diff
@@ -3627,7 +3627,7 @@ test(1149.2, forderv(numeric(0)), integer(0))
 
 # test uniqlengths
 set.seed(45)
-x <- sample(c(NA_integer_, 1:1e5), 1e7, TRUE)
+x <- sample(c(NA_integer_, 1:1e4), 1e6, TRUE)
```

Comment:
old test, not possible to git reset to test validity of smaller size

Reply:
Good. Looks fine to me, too, to be smaller. Now that CRAN_Release.cmd has --strict-barier together with ASAN, and have seen that work and detect problems now that were missed before, I'm much confident we don't need to "go big" in tests like this.